WORKSPACE.bazel

```diff
@@ -1008,3 +1008,12 @@ http_file(
     sha256 = "454891cac2421f3f894759ec5e6b6e48fbb544d79197bc29b88d34b93d78a4f1",
     url = "https://download.dfinity.systems/ic/52ebccfba8855e23dcad9657a8d6e6be01df71f9/binaries/x86_64-linux/pocket-ic.gz",
 )
+
+# IC-OS deps
+http_file(
```

Comment:
I'm adding the Ubuntu base image here to be used by the test, it will be used by other rules (all IC-OS builds) as well.